packages/cli/src/handlers/login.ts

```diff
@@ -102,28 +99,18 @@ const loginWithPassword = async (url: string) => {
     }
     const { userUuid, organizationUuid } = loginBody.results;
     const cookie = header.split(';')[0].split('=')[1];
-    const patUrl = new URL(`/api/v1/user/me/personal-access-tokens`, url).href;
```

Comment:
logic to create a PAT extracted into a separate method, to be used in : 
- generate PAT after user/password login
- generate PAT after oauth login